### PR TITLE
fix(SUP-18895): Vertical Space is cutting Thai letters

### PR DESCRIPTION
**Pull request by shalommeoded**
Added additional line spacing for the default Closed Captioning style in order to support Thai Characters.

### CODE REVIEW

**Comment by OrenMe** [comment]
@yairans PTAL

File: modules/KalturaSupport/resources/mw.ClosedCaptions.js [hunk 1]
```diff
@@ -1014,7 +1014,8 @@
 		},
 		getDefaultCaptionCss: function(){
 			var style = {};
-			style["display"] = "inline";
+			style["display"] = "inline-block";
+			style["line-height"] = "2";
```
Comment (yairans) [inline modules/KalturaSupport/resources/mw.ClosedCaptions.js]: @shalommeoded from my test [here](http://www.kaltura.com/index.php/extwidget/preview/partner_id/2290881/uiconf_id/39595521/entry_id/1_epi34r91/embed/dynamic?debugKalturaPlayer) `dispaly:inline-block` is enough. no?

Comment (shalommeoded) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: @yairans  in some cases, Thai letters can have up to 3-4 additional symbols on top of the character, therefore, I have added a bit more line space to the background.

Comment (yairans) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: again, as is sow `display:inline-block` solves the issue. did you see else?

Comment (shalommeoded) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: Yes, in Thai letters, the background will cut in some cases the Thai symbols. If you can see that `display:inline-block` is enough I can alter the fix.

Comment (yairans) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: This is what I see but better you verify by yourself

Comment (shalommeoded) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: @yairans 
Just checked, you are right, the difference does not justify to enhance the line-height.

**Pull request merged**